src/modules/swap/services/safeBundleFlow/safeBundleEthFlow.ts

```diff
@@ -124,13 +124,10 @@ export async function safeBundleEthFlow(
     )
     tradeFlowAnalytics.sign(swapFlowAnalyticsContext)
 
-    logTradeFlow(LOG_PREFIX, 'STEP 8: add app data to upload queue')
```

Comment:
we don't need to upload the app-data because we already do `signAndPostOrder` which upload it anyways